eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -36,10 +38,12 @@
           "netcoreapp21_Coverage_Record": {
               "SupportsRecording": true,
               "CollectCoverage": true,
-              "TestTargetFramework": "netcoreapp2.1"
+              "TestTargetFramework": "netcoreapp2.1",
+              "Runtime": "win-x64"
           },
           "net5.0": {
-              "TestTargetFramework": "net5.0"
+              "TestTargetFramework": "net5.0",
+              "Runtime": "win-x64"
```

Comment:
You don't need to add it here, this gets combined with the Agent config above.

Reply:
good point.